tests/conditional_compilation/test_size_tracking.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+import pytest
+
+from path_utils import library_name # pylint: disable=import-error
+
+
+@pytest.fixture(scope="session")
+def conditional_compilation_libs_name():
+    """ Fixture function for getting lib names with specific extension"""
+    libs_name = ['inference_engine_transformations', 'MKLDNNPlugin', 'ngraph']
+    return [library_name(lib_name) for lib_name in libs_name]
+
+
+@pytest.fixture(scope="session")
+def ref_libs_size(openvino_root_dir, conditional_compilation_libs_name):
+    """Fixture function for getting lib sizes in """
+    return get_lib_sizes(openvino_root_dir, conditional_compilation_libs_name)
+
+
+def get_lib_sizes(path, lib_names):
+    """Function for getting lib sizes by lib names"""
+    assert Path.exists(path), f'Directory {path} isn\'t created'
+    return {lib: Path(path).joinpath(lib).stat().st_size for lib in lib_names}
+
+
+def test_size_tracking_libs(ref_libs_size, conditional_compilation_libs_name, test_id, model, artifacts):
+    install_prefix = artifacts / test_id / 'install_pkg'
+    lib_sizes = get_lib_sizes(install_prefix, conditional_compilation_libs_name)
+    res = [key for key in lib_sizes.keys() if lib_sizes[key] > ref_libs_size[key]]
```

Comment:
Can we introduce the reference sizes for libraries and also check that the size decrease should be around ...Kb?

Reply:
I added check decrease.
Please see if that is what you mean?